tests/roles/neutron_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,36 @@
+- name: deploy podified Neutron
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc patch openstackcontrolplane openstack --type=merge --patch '
+    spec:
+      neutron:
+        enabled: true
+        template:
+          databaseInstance: openstack
+          containerImage: quay.io/podified-antelope-centos9/openstack-neutron-server:current-podified
```

Comment:
I think we want to avoid specifying the image. For CI purposes we control this via kustomize, the image should already be set at this point even though the service is disabled.